molecule/default/molecule.yml

```diff
@@ -6,8 +6,8 @@ driver:
 lint:
   name: yamllint
 platforms:
-  - name: amazon2018.03
-    image: amazonlinux:2018.03
+  - name: amazonlinux2
```

Comment:
Same question.

Reply:
These are Docker images, not AMIs.  See [here](https://hub.docker.com/_/amazonlinux/?tab=description) for more details.